client/packages/grandpa-ranger/src/index.ts

```diff
@@ -0,0 +1,138 @@
+import {generateRange} from "./collect";
+
+require('dotenv').config()
+import { Connection } from './connection';
+import { cryptoWaitReady } from "@t3rn/sdk"
+import { Prometheus } from "./prometheus";
+
+class GrandpaRanger {
+	circuit: Connection;
+	target: Connection;
+	config: any;
+	prometheus: Prometheus;
+
+	constructor(config: any) {
+		this.config = config;
+		this.prometheus = new Prometheus();
+		this.prometheus.rangeBreak.inc(this.config.rangeBreak);
```

Comment:
maybe rangeInterval?

Reply:
much better. My creativity for variable naming is not great haha